pkg/driver/controller.go

```diff
@@ -330,6 +339,8 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 }
 
 func (d *Driver) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequest) (*csi.DeleteVolumeResponse, error) {
+
+	_ = co.StripSecretsAndGetChapParams(req)
```

Comment:
hmm, is this doing anything?

Reply:
Yes, stripping the secrets only. The chap parameters returned are discarded. It is not needed for DeleteVolume case.